.eslintrc.json

```diff
@@ -19,8 +19,9 @@
     "@typescript-eslint/no-unsafe-member-access": 0,
     "@typescript-eslint/no-unsafe-return": 0,
     "@typescript-eslint/no-unsafe-call": 0,
-    "react/display-name": 0
-
+    "react/display-name": 0,
+    "react-hooks/exhaustive-deps": "off",
+    "react-hooks/rules-of-hooks": "off"
```

Comment:
I turned the rules-of-hooks on and ran lint with no errors. Why do you think we want this turned off?